content/blog/eng/v-protetto17-5-2024.md

```diff
@@ -0,0 +1,119 @@
+---
+
+authors: ["mgarza", "ana-radujko"]
+comments: true
+date: "2024-03-11 17:00:00"
+draft: false
+share: true
+categories: ['Protected Fridays']
+title: "Venerdì Protetto | May 2024 edition"
+description: "What happened in the last issue of Venerdì Protetto?"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/social/social-preview-vp.png'
+twitterLargeImage: '/images/social/social-preview-vp.png'
+toc: false
+
+---
+
+
+Topics:
+
+
+During the last Venerdì Protetto (held on May 17, 2024), we had both a talk and a workshop. There they are:
+
+- [Hack the Box Workshop](#hack-the-box-workshop) by [Alessio Giorgianni](https://www.linkedin.com/in/alessio-giorgianni-b90500123/) (Facile.it)
+- [How to develop cognitive flexibility?](#how-to-develop-cognitive-flexibility) by [Arduino Mancini](https://www.linkedin.com/in/arduinomancini/) ([TIBICON](https://www.tibicon.net/))
+
+
+# Hack the box Workshop
```

Comment:
Io ho sempre usato ## per i titoli e ### per i sottotitoli. Non è obbligatorio, vedi tu.

Reply:
Avevo fatto così perché l'articolo di Alessio è gigante, e quello di Arduino si perdeva, ma se lo estrapoliamo in un altro articolo posso rimettere i titoli con il livello indicato dalla tua guida